### PR TITLE
Update Remote Config plugin

### PR DESCRIPTION
**Pull request by kroikie**
- Bump Android version
- iOS returns String status

### CODE REVIEW

File: packages/firebase_remote_config/android/build.gradle [hunk 1]
```diff
@@ -32,7 +32,6 @@ android {
         disable 'InvalidPackage'
     }
     dependencies {
-        api 'com.google.firebase:firebase-config:[11.4.0,12.0['
-        api 'com.google.firebase:firebase-core:[11.4.0,12.0['
+        api 'com.google.firebase:firebase-config:15+'
```
Comment (collinjackson) [inline packages/firebase_remote_config/android/build.gradle]: Can you elaborate on why this is necessary, just for my own edification?

File: packages/firebase_remote_config/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+## [0.0.2]
+
+* Fix iOS casting issue.
```
Comment (collinjackson) [inline packages/firebase_remote_config/CHANGELOG.md]: This is a bit vague, we might want to describe the APIs or behavior that was previously broken and now works

File: packages/firebase_remote_config/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
...
+* Bump Android library version.
```
Comment (collinjackson) [inline packages/firebase_remote_config/CHANGELOG.md]: I would say something like "increase Android Firebase dependency to 15+" to be more specific about what you did here

File: packages/firebase_remote_config/android/build.gradle (hunk 1 above)
Comment (kroikie) [reply packages/firebase_remote_config/android/build.gradle]: All libraries from Google Play Services must be of the same version if used in the same app.  We should devise a way to auto update all the plugins so they stay in sync.

File: packages/firebase_remote_config/CHANGELOG.md (hunk 2 above)
Comment (kroikie) [reply packages/firebase_remote_config/CHANGELOG.md]: Done

File: packages/firebase_remote_config/CHANGELOG.md (hunk 3 above)
Comment (kroikie) [reply packages/firebase_remote_config/CHANGELOG.md]: Done.

File: packages/firebase_remote_config/android/build.gradle (hunk 1 above)
Comment (bjon) [reply packages/firebase_remote_config/android/build.gradle]: i mean `15.+`, not `15+`.

Comment (kroikie) [reply packages/firebase_remote_config/android/build.gradle]: Good Catch!

**Comment by collinjackson** [comment]
lgtm

**Pull request merged**